workflows/short-read-mngs/host_filter.wdl

```diff
@@ -175,7 +176,7 @@ workflow czid_host_filter {
 
     File kallisto_transcript_abundance_tsv = kallisto.transcript_abundance_tsv
     File kallisto_ERCC_counts_tsv = kallisto.ERCC_counts_tsv
-    File? kallisto_gene_abundance_tsv = kallisto.gene_abundance_tsv
+    File kallisto_transcript_gene_mapping_tsv = kallisto.transcript_gene_mapping_tsv
```

Comment:
Is this an optional file? Will this be produced on every pipeline run?

Reply:
I don't think it's an optional file, because the `kallisto.transcript_abundance_tsv` is produced on every run as far as I can tell, and `kallisto.transcript_gene_mapping_tsv` is a direct child of that file that we want to produce for every transcript TSV.

Reply:
ok make sense, thanks!